ni/measurements/metadata/v1/metadata_store_service.proto

```diff
@@ -57,118 +57,123 @@ service MetadataStoreService {
   // Creates a new test station in the metadata store.
   rpc CreateTestStation(CreateTestStationRequest) returns (CreateTestStationResponse);
 
-  // Gets the hardware associated with the identifier given in the request.
-  rpc GetHardware(GetHardwareRequest) returns (GetHardwareResponse);
+  // Gets the hardware item associated with the identifier given in the request.
+  rpc GetHardwareItem(GetHardwareItemRequest) returns (GetHardwareItemResponse);
   // Perform an OData query
-  rpc QueryHardware(QueryHardwareRequest) returns (QueryHardwareResponse);
-  // Creates a new hardware in the metadata store.
-  rpc CreateHardware(CreateHardwareRequest) returns (CreateHardwareResponse);
+  rpc QueryHardwareItems(QueryHardwareItemsRequest) returns (QueryHardwareItemsResponse);
+  // Creates a new hardware item in the metadata store.
+  rpc CreateHardwareItem(CreateHardwareItemRequest) returns (CreateHardwareItemResponse);
 
-  // Gets the software associated with the identifier given in the request.
-  rpc GetSoftware(GetSoftwareRequest) returns (GetSoftwareResponse);
+  // Gets the software item associated with the identifier given in the request.
+  rpc GetSoftwareItem(GetSoftwareItemRequest) returns (GetSoftwareItemResponse);
   // Perform an OData query
-  rpc QuerySoftware(QuerySoftwareRequest) returns (QuerySoftwareResponse);
-  // Creates a new software in the metadata store.
-  rpc CreateSoftware(CreateSoftwareRequest) returns (CreateSoftwareResponse);
+  rpc QuerySoftwareItems(QuerySoftwareItemsRequest) returns (QuerySoftwareItemsResponse);
+  // Creates a new software item in the metadata store.
+  rpc CreateSoftwareItem(CreateSoftwareItemRequest) returns (CreateSoftwareItemResponse);
+
+  // Gets the test adapter associated with the identifier given in the request.
+  rpc GetTestAdapter(GetTestAdapterRequest) returns (GetTestAdapterResponse);
+  // Perform an OData query
+  rpc QueryTestAdapters(QueryTestAdaptersRequest) returns (QueryTestAdaptersResponse);
+  // Creates a new test adapter in the metadata store.
+  rpc CreateTestAdapter(CreateTestAdapterRequest) returns (CreateTestAdapterResponse);
 
   // Registers a schema. Once a schema has been published, it cannot be modified or removed.
-  rpc RegisterMetadataSchema(RegisterMetadataSchemaRequest) returns (RegisterMetadataSchemaResponse);
-  // Enumerate the metadata schemas that have been previously registered
-  rpc EnumerateMetadataSchemas(EnumerateMetadataSchemasRequest) returns (EnumerateMetadataSchemasResponse);
+  rpc RegisterExtensionSchema(RegisterExtensionSchemaRequest) returns (RegisterExtensionSchemaResponse);
+  // List the extension schemas that have been previously registered
+  rpc ListExtensionSchemas(ListExtensionSchemasRequest) returns (ListExtensionSchemasResponse);
 
   // Resolves a given alias to its target (i.e., the underlying metadata that it represents)
   rpc ResolveAlias(ResolveAliasRequest) returns (ResolveAliasResponse);
```

Comment:
Should this just be `GetAlias` or is there a different semantic here that I missed for resolving? I guess maybe one difference is that all of the other get methods take an id where this takes a name. However, if you end up adding an id field to all of these resource messages then it seems more or less identical unless you want to still allow lookups based on the alias name rather than alias id.

Reply:
For consistency, I don't have any problem renaming this to `Get`.